src/main/java/com/github/maven_nar/Msvc.java

```diff
@@ -164,6 +170,12 @@ public void configureLinker(final LinkerDef linker) throws MojoExecutionExceptio
         for (File sdkLib : sdkLibs)
           linker.addLibraryDirectory(sdkLib, sdkArch);
 
+      final String envLib = System.getenv("LIB");
```

Comment:
`LIBPATH` might be a better choice, indeed, unless you want to ignore all environment variables.